docs/rollkit-stack.md

```diff
@@ -1,30 +1,26 @@
 ---
 sidebar_position: 3
 sidebar_label: Rollkit Stack
-description: Intro to Rollkit's stack, a modular development kit for building rollup chains.
+description: Intro to Rollkit's stack, a modular rollup framework.
 ---
 
 # The Rollkit Stack
 
 This section will cover the technical stack of Rollkit.
 
-Rollkit is built by replacing Tendermint, the Cosmos-SDK Consensus Layer,
-with a drop-in replacement that communicates directly with Celestia's Data
-Availability (DA) and Consensus Layer. Our version of Tendermint is designed
-to work seamlessly with other modular layers, allowing for greater flexibility
-and adaptability.
-
-It spins up a sovereign rollup, which collects transactions into blocks and
+It spins up a rollup, which collects transactions into blocks and
 posts them onto Celestia for DA and Consensus.
 
-The goal of Rollkit is to enable anyone to design and deploy a sovereign
-rollup on Celestia in minutes with minimal overhead.
+The goal of Rollkit is to enable anyone to design and deploy a
+rollup on any DA layer with minimal overhead. Rollkit aims to be a neutral
```

Comment:
credibly neutral

Reply:
resolved in https://github.com/rollkit/docs/pull/57/commits/eb6729eb1217ff7d2bea3d586c0909ae5032c99e